web/src/components/tasks/business-formation/review/ReviewStep.tsx

```diff
@@ -76,17 +85,12 @@ export const ReviewStep = (): ReactElement => {
               </ReviewSubSection>
             </>
           )}
-          {!isNonProfit && hasProvisions && <ReviewProvisions />}
-          {!isNonProfit && hasPurpose && (
-            <ReviewSubSection header={Config.formation.fields.businessPurpose.label}>
-              <ReviewText fieldName={"businessPurpose"} isExpandable={true} />
-            </ReviewSubSection>
-          )}
+          {!isNonProfit && getProvisionsAndPurposeSections()}
         </ReviewSection>
         <ReviewSection stepName={"Contacts"} testId="edit-contacts-step">
           <ReviewRegisteredAgent />
-          {(isNonProfit || isCorp || (!isLP && hasMembers)) && <ReviewMembers />}
-          {<ReviewSignatures />}
+          {(isCorp || (!isLP && hasMembers)) && <ReviewMembers />}
```

Comment:
why did we get rid of isNonProfit here?

Reply:
Because `isCorp` includes both `nonprofit` and `foreign-nonprofit`. I meant to call that out directly.

Reply:
got it!

Reply:
@jphechter it does?? That's not what I'm seeing
![image](https://github.com/newjersey/navigator.business.nj.gov/assets/6563663/e950e147-a5bd-4545-869d-16af282838c7)

Reply:
@aloverso @cbrunefearless Whoops. Y'all are right.

I hovered over `corpLegalStructures` and VS Code lied to me.
![Screenshot 2023-08-24 at 10 15 29 AM](https://github.com/newjersey/navigator.business.nj.gov/assets/22647707/a5385988-7a84-498e-8f91-4d29533d51fa)

It's not entirely wrong, but I guess I'm getting that because the variable is of type `FormationLegalType[]` which is technically all the formation legal types.

I'll fix this.

Reply:
yeah, that's the type definition, not the actual value of the array

Reply:
@jphechter tbh I'm concerned that there wasn't a failing test here, shouldn't something be failing if we removed the `ReviewMembers` component for nonprofits (if i'm understanding this logic correctly)? Can you investigate and maybe add test coverage for this?

Reply:
added!